docs/CHANGELOG.rst

```diff
@@ -44,91 +44,97 @@ and anyone can contribute to it.
 It is possible to perform full-scale analyses and contribute to the production
 of CTA performances for different science cases or perform studies that require big amounts of data.
 
+
 Contributors
 ++++++++++++
 
-- Michele Peresano (`@HealthyPear`_)
+@HealthyPear and @kosack
 
 What is changed since v0.4.0.post1
 ++++++++++++++++++++++++++++++++++
 
 Pull-requests containing changes of multiple nature are repeated.
 
-🚀 General features
-^^^^^^^^^^^^^^^^^^^
+- fixed missing import of QTable (#188) @kosack
+- try to fix build (mamba error) (#189) @kosack
+- Add benchmarking notebook template (#181) @HealthyPear
 
-- Add book template material (:pr:`174`) `@HealthyPear`_
-- Add new script for benchmarking (:pr:`157`) `@HealthyPear`_
-- Update the entire benchmark suite (:pr:`166`) `@HealthyPear`_
-- Improve data scripts output and update analysis default settings (:pr:`165`) `@HealthyPear`_
-- Improve protopipe.mva and protopipe-MODELS (:pr:`164`) `@HealthyPear`_
-- Add image extraction status for TwoPassWindowSum (:pr:`163`) `@HealthyPear`_
-- Add support for prod5N files (:pr:`167`) `@HealthyPear`_
-- Add I/O functions to simplify benchmarking (:pr:`147`) `@HealthyPear`_
-- Apply CALIB\_SCALE directly from ctapipe.io.SimtelEventSource (:pr:`145`) `@HealthyPear`_
-- Add progress bar to both TRAINING and DL2 production scripts (:pr:`150`) `@HealthyPear`_
-- Add option to enable/disable charge integration integration correction  (:pr:`146`) `@HealthyPear`_
-- Calculate impact parameter always in the tilted frame (:pr:`143`) `@HealthyPear`_
-- Add support for calibscale (:pr:`140`) `@HealthyPear`_
-- Add optional LST stereo trigger requirement (:pr:`131`) `@HealthyPear`_
-- Update to ctapipe 0.11.0 (:pr:`136`) `@HealthyPear`_
-- Add configuration option to choose which cleaning to use to get training data (:pr:`135`) `@HealthyPear`_
-- Add choice of estimation weigths and standard deviation for RandomForestRegressor models (:pr:`134`) `@HealthyPear`_
-- Add concentration (:pr:`133`) `@HealthyPear`_
-- Improve models generation (:pr:`96`) `@HealthyPear`_
-- Support for ctapipe 0.10.5 (:pr:`124`) `@HealthyPear`_
-- Small improvements to modeling script (:pr:`114`) `@HealthyPear`_
+🚀 General features
+^^^^^^^^^^^^^^^^^^^^
+
+- Add book template material (:pr:`#174`) @HealthyPear
```

Comment:
all users should be cited as e.g. `@HealthyPear`_ in order for their links to work for contact

Reply:
or maybe we can leave the links to the Contributors sections where the full names should also be listed like I was doing